website/docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -199,7 +199,11 @@ options:
     menu_type: chisel3
     nested_options:
       - title: Latest
-        url: api/chisel3/latest/
+        url: api/chisel/latest/
+      - title: 6.0
+        url: api/chisel/6.0/
```

Comment:
I'd been thinking... do we need to do the redirect here or could we do an explicit `https://javadoc.io/doc/org.chipsalliance/chisel_2.13/6.0` ?

Reply:
We don't need it but it keeps the link to javadoc.io in just 1 place, a bit DRYer